cfn-templates/cid-cfn.yml

```
@@ -655,17 +655,21 @@ Resources:
                       else:
                           return (True, f'ERROR: error invoking crawler {CRAWLER} {exc}')
                   print('started crawler started. waiting for crawler to finish')
-                  start_time = time.time()
-                  while time.time() - start_time < timeout_seconds:
-                      time.sleep(10)
-                      crawler_status = glue.get_crawler(Name=CRAWLER)['Crawler']['State']
-                      print('status=', crawler_status)
-                      if crawler_status in ('READY', 'STOPPING'):
-                          print("Stop waiting")
-                          break
-                  else:
-                      return (True, f"Timeout exceeded. Crawler '{CRAWLER}' did not complete. This is not a fatal error and the rest of the deployment will continue.")
+                  try:
+                      start_time = time.time()
+                      while time.time() - start_time < timeout_seconds:
+                          time.sleep(10)
```

why 10 s?

---

average crawler runs about 60 s 

we can change the wait to 1 not an issue

```suggestion
                          time.sleep(1)
```